src/disco/api.js

```diff
@@ -12,10 +12,10 @@ export function getDiscoveryAddons({ api, taarParams = {}, _config = config }) {
     return object;
   }, {});
 
-  // We translate `clientId` to `'telemetry-client-id'`.
-  if (allowedTaarParams.clientId) {
-    allowedTaarParams['telemetry-client-id'] = allowedTaarParams.clientId;
-    delete allowedTaarParams.clientId;
+  // We translate `taarId` to `'telemetry-client-id'`.
```

Comment:
As per my suggestion above, I would not change that. That would reduce the diff in this file.